Core/System.cpp

```diff
@@ -593,6 +593,8 @@ std::string GetSysDirectory(PSPDirectories directoryType) {
 			return g_Config.appCacheDirectory;
 		}
 		return g_Config.memStickDirectory + "PSP/SYSTEM/CACHE/";
+	case DIRECTORY_VIDEO_DUMP:
+		return g_Config.memStickDirectory + "PSP/FRAMEDUMP/";
```

Comment:
Wasn't too sure where to place the dumps, but I guess placing them into VIDEO is fine.